tests/test_process.py

```diff
@@ -42,31 +13,42 @@ def test_build_url(thredds_base, filepath, variable, lat, lon):
             "https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thredds/dodsC/datasets",
             "postgresql://httpd_meta@db3.pcic.uvic.ca/pcic_meta",
             "tasmax_day_BCCAQv2_bcc-csm1-1-m_historical-rcp26_r1i1p1_19500101-21001231_Canada",
-            "[91:91]",
-            "[206:206]",
+            "[0:1:91]",
+            "[0:1:206]",
         )
     ],
 )
 @pytest.mark.parametrize(
     ("variable"),
     [
-        "tasmax[0:1:55114]",
-        "tasmax[0:55114]",
+        "tasmax[0:1:0]",
+        "tasmax[0:1:15000]",
     ],
 )
 def test_process(thredds_base, connection_string, unique_id, variable, lat, lon):
-    with NamedTemporaryFile(
-        suffix=".nc", prefix="output_", dir="/tmp", delete=True, mode="w+"
-    ) as out_file:
-        data_output = process_request(
-            connection_string,
-            unique_id,
-            thredds_base,
-            variable,
-            lat,
-            lon,
-            out_file.name,
-        )
-        filepath = find_filepath(connection_string, unique_id)
-        url = build_url(thredds_base, filepath, variable, lat, lon)
-        assert open_dataset(url) == open_dataset(data_output)
+    data_files = process.process_request(
+        connection_string,
+        unique_id,
+        thredds_base,
+        variable,
+        lat,
+        lon,
+    )
+
+    # Prep expected output
+    filepath = finder.find_filepath(connection_string, unique_id)
+    url = request_opendap.build_url(thredds_base, filepath, variable, lat, lon)
+    data = open_dataset(url)
+    split_urls = split.split_url(url, data.nbytes)
```

Comment:
I feel like this could be a helper method. We may need it elsewhere in the tests.